apps/meeting/src/providers/AppStateProvider.tsx

```diff
@@ -15,13 +15,17 @@ interface AppStateValue {
   region: string;
   isWebAudioEnabled: boolean;
   blurOption: string,
+  isEchoReductionEnabled: boolean;
   meetingMode: MeetingMode;
   layout: Layout;
+  joinInfo: any;
```

Comment:
Avoid `any` if possible.

Reply:
keep the same from JS SDK. https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts#L310

Reply:
JS SDK demo doesn't use typescript. But since we are using typescript in react meeting demo. We should avoid `any`.